dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -185,9 +185,17 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
         if len(output_column_order) == 1 and output_column_order[0] == "UInt8(1)":
             backend_output_column_order = [df_agg.columns[0]]
         else:
+
+            def try_get_backend_by_frontend_name(oc):
+                try:
+                    return cc.get_backend_by_frontend_name(oc)
+                except KeyError:
+                    return oc
+
```

Comment:
do you think this is worth moving to the column_container class? or do you think this is a one off case needed only here?

Reply:
I think this is a pretty one off case